nautobot/docs/user-guides/relationships.md

```diff
@@ -0,0 +1,108 @@
+
+For more technical details on how to use this feature, please see the documentation on [Relationships](../models/extras/relationship.md).
+
+## Relationship details
+
+This table defines repository parameters that are required to establish a relationship.
+
+|Field|Explanation|
+|:--|:--|
+|Name|Name of the relationship as displayed to users.|
+|Slug|Internal name of this relationship. Please use underscores rather than dashes.|
+|Description|(Optional) Meaningful description to provide additional relationship details.|
+|[Type](../models/extras/relationship.md#relationship-types)|Cardinality of this relationship.|
+|Source type|The source object type to which this relationship applies.|
+|Source Label|(Optional) Label for related destination objects, as displayed on the source object.|
+|Source filter|(Optional) Filterset filter matching the applicable source objects of the selected type.|
+|Destination type|The destination object type to which this relationship applies.|
+|Destination Label|(Optional) Label for related destination objects, as displayed on the destination object.|
+|Destination filter|(Optional) Filterset filter matching the applicable destination objects of the selected type.|
+
+As seen in the table above relationships offers many additional optional options for more advanced functionality, those options will be discussed further during this user guide.
+
+## Using a Relationship
+
+This section will focus on examples relationships.
+
+### Adding a Relationship
+
+To create a relationship, navigate to **Extensibility > Data Management > Relationships**
+
+A simple relationship between `Device to Vlan` which is a `many-to-many` relationship. This relationship is `many-to-many` because more than one Devices can be assigned more than one Vlans. This relationship object is shown below.
+
+![many-to-many example](./images/relationships/01-relationships.png)
+
+### Extensibility of a Relationship
+
+Now that this relationship is created we can navigate over to a Device detail view and we see `VLANs` is now listed under the `Relationships` section.
+
+![device-to-vlan relationship](./images/relationships/02-relationships.png)
+
+What if your business requirements or terminology differs from what you want? For example lets say you want `VLANs` to be called `VIDs`, you can simply update the relationship `source label` field based on your needs.
+
+You can see now the relationship name is `VIDs` instead of `VLANs`.
+
+![updated-source-label](./images/relationships/03-relationships.png)
+
+!!! info
+    In a similar fashion the destination label can be changed to meet business requirements.
+
+Next, some of the visibility options will be discussed. The first option to be demonstrated is the ability to `Move to Advanced tab`, in some views the user may want the relationship data to be displayed on the `Advanced` tab instead of within the detail view of the object.
```

Comment:
Perhaps:

```suggestion
Next, some of the visibility options will be discussed. The first option to be demonstrated is the ability to `Move to Advanced tab`. In some cases a particular relationship may not be relevant to the typical user (maybe it's used exclusively in automation) and so you may want the relationship data to be displayed on the `Advanced` tab instead of within the default detail view of the object.
```